### PR TITLE
[SampleSQLBuilder-default-service-account-user]Lpd 33994

### PR DESCRIPTION
**Pull request by slnn**
https://liferay.atlassian.net/browse/LPD-33994

### CODE REVIEW

**Comment by slnn** [comment]
ci:test:relevant

File: modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java [hunk 1]
```diff
@@ -5569,8 +5575,9 @@ public UserModel newSampleUserModel() {
 		_sampleUserId = _counter.get();
 
 		return newUserModel(
-			_sampleUserId, _SAMPLE_USER_NAME, _SAMPLE_USER_NAME,
-			_SAMPLE_USER_NAME, UserConstants.TYPE_REGULAR);
+			_counter.get(), "default-service-account",
```
Comment (dantewang) [inline modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java:5578]: @slnn Why do you change the `newSampleUserModel()` to generate the `default-service-account`?

Comment (slnn) [reply modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java:5578]: Sorry, my bad, will resend

**Comment by slnn** [comment]
Resend in https://github.com/liferay-core-infra/liferay-portal/pull/7348

**Pull request closed without merging**